packages/google_identity_services_web/example/id/lib/main.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: avoid_print
+
+import 'package:google_identity_services_web/id.dart' as id;
+import 'package:google_identity_services_web/loader.dart' as gis;
+import 'package:js/js.dart' show allowInterop;
+import 'package:jwt_decoder/jwt_decoder.dart' as jwt;
+
+void main() async {
+  await gis.loadWebSdk(); // Load the GIS SDK
+
+  id.setLogLevel('debug');
+
+  final id.IdConfiguration config = id.IdConfiguration(
+    client_id: 'your-client_id.apps.googleusercontent.com',
+    ux_mode: id.UxMode.popup,
+    callback: allowInterop(onCredentialResponse),
+  );
+
+  id.initialize(config);
+  id.prompt(allowInterop(onPromptMoment));
+  //runApp(const MyApp());
```

Comment:
This file still has the commented out runApp and the empty doc comments on methods.

Reply:
Cleaned up